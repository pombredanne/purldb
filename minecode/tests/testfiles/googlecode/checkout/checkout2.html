


<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"projectHomeUrl": "/p/ossbuild", "loggedInUserEmail": null, "profileUrl": null, "assetHostPath": "http://www.gstatic.com/codesite/ph", "projectName": "ossbuild", "token": null, "assetVersionPath": "http://www.gstatic.com/codesite/ph/11512667522589661435", "domainName": null, "relativeBaseUrl": ""};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 _gaq.push(
 ['projectTracker._setAccount', 'UA-12802436-1'],
 ['projectTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>Source Checkout - 
 ossbuild -
 
 
 Open Source Software Build Environment - Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/ph_detail.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
h4 {
 margin-top: 10px;
 margin-bottom: 10px;
}
.indent {
 margin-left: 2.3em;
 margin-bottom: 2em;
}
.bubble_wrapper {
 display: table;
}
.bubble {
 padding: 0.8em;
}
#quota {
 margin-top: 2.5em;
}

 
 </style>
</head>
<body class="t4">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fp%2Fossbuild%2Fsource%2Fcheckout&amp;followup=http%3A%2F%2Fcode.google.com%2Fp%2Fossbuild%2Fsource%2Fcheckout" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 itemscope itemtype="http://schema.org/CreativeWork">
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="/p/ossbuild">
 <a href="/p/ossbuild/">
 
 
 <img src="/p/ossbuild/logo?cct=1270773548"
 alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/p/ossbuild/"><span itemprop="name">ossbuild</span></a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link"
 href="/p/ossbuild/"><span itemprop="description">Open Source Software Build Environment</span></a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/hosting/search">
 <input size="30" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/p/ossbuild/" class="tab ">Project&nbsp;Home</a>
 
 
 
 
 <a href="/p/ossbuild/downloads/list" class="tab ">Downloads</a>
 
 
 
 
 
 <a href="/p/ossbuild/w/list" class="tab ">Wiki</a>
 
 
 
 
 
 <a href="/p/ossbuild/issues/list"
 class="tab ">Issues</a>
 
 
 
 
 
 <a href="/p/ossbuild/source/checkout"
 class="tab active">Source</a>
 
 
 
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 <td class="subt">
 <div class="st1">
 <div class="isf">
 
 


 <span class="inst1"><a href="/p/ossbuild/source/checkout">Checkout</a></span> &nbsp;
 <span class="inst2"><a href="/p/ossbuild/source/browse/trunk">Browse</a></span> &nbsp;
 <span class="inst3"><a href="/p/ossbuild/source/list">Changes</a></span> &nbsp;
 
 
 
 
 
 
 
 </form>
 <script type="text/javascript">
 
 function codesearchQuery(form) {
 var query = document.getElementById('q').value;
 if (query) { form.action += '%20' + query; }
 }
 </script>
 </div>
</div>

 </td>
 
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
 
 style="padding:0 3px 3px 0; margin:0"
 
 
>


<div style="margin-bottom:.4em">
 <table align="center" border="0" cellspacing="0" cellpadding="0" id="cue">
 <tr><td class="cuetip">
 
 <div class="box-inner" style="white-space:nowrap; padding-top:2px; font-size: 92%;">
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 <b>How-to:</b>
 Explore this project's source code by clicking the "Browse" and "Changes" links above.
 
 
 
 
 
 
 
 
 </div>
 <div class="round1"></div>
 <div class="round2"></div>
 <div class="round4"></div>
 </td></tr>
 </table>
</div>


 





 
 



<div style="padding: 10px">
<h4>Command-line access</h4>
<div class="indent">
 
 
 <p>Use this command to anonymously check out the latest project source code:</p>
 <div class="bubble_wrapper">
 <div class="pmeta_bubble_bg">
 <div class="round4"></div>
 <div class="round2"></div>
 <div class="round1"></div>
 <div class="box-inner">
 <tt># Non-members may check out a read-only working copy anonymously over HTTP.</tt><br>
 <tt id="checkoutcmd">svn checkout <strong><em>http</em></strong>://ossbuild.googlecode.com/svn/trunk/ ossbuild-read-only</tt>
 </div>
 <div class="round1"></div>
 <div class="round2"></div>
 <div class="round4"></div>
 </div>
 </div>
 
</div>
<h4>GUI and IDE access</h4>
<div class="indent">
 <p>This project's Subversion repository may be accessed using many different
 <a href="http://subversion.apache.org/packages.html">client programs and plug-ins</a>.
 See your client's documentation for more information.</p>
</div>

</div>





 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 </body>
</html>

