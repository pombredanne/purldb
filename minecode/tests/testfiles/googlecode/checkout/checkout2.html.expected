[{'vcs_repository': u'http://ossbuild.googlecode.com/svn/',
  'vcs_tool': u'svn'}]